controllers/informationCenter/InformationCenterHandler.inc.php

```diff
@@ -153,7 +154,8 @@ public function _listNotes($args, $request)
 
         $templateMgr = TemplateManager::getManager($request);
         $noteDao = DAORegistry::getDAO('NoteDAO'); /** @var NoteDAO $noteDao */
-        $templateMgr->assign('notes', $noteDao->getByAssoc($this->_getAssocType(), $this->_getAssocId()));
+        $notes = $noteDao->getByAssoc($this->_getAssocType(), $this->_getAssocId())->toArray();
+        $templateMgr->assign('notes', new ArrayItemIterator($notes));
```

Comment:
Rather than using `ArrayItemIterator`, I'd recommend converting the `{iterate}` calls in the template to regular `{foreach}`.

Reply:
(You would also be able to remove the `toArray` call above and just let the `LazyCollection` go to the template.)

Reply:
Changed.